ansible/kria.md

```diff
@@ -240,6 +242,8 @@ By default Note Division Sync is off, Division Cueing is off, and Division Sync
 
 ### Config
 
+![](../images/grid_KR_config.png)
```

Comment:
@mrmattallison : this image seems to have a few loose indicators + unmatched text. wanna give it another once-over? <3

Reply:
Hey @dndrks not loose, I just have no idea what the heck those buttons do! I asked @csboling to chime in when I pushed the update. I’d anyone can tell me I’ll add them in.